src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -133,9 +133,22 @@
 --    add created* & last_modified* to transcription_status
 --    add tablespace clause to transcription_status & transcription_type
 --    add trigger and associated function to transcription_workflow
+--v50 rename device_register to node_register, and column device_type to node_type
+--    remove last_modifed* from COURTROOM, EVENT_HANDLER, NODE_REGISTER
+--    remove created* & last_modified* from EXTERNAL_LOCATION_TYPE, OBJECT_DIRECTORY_STATUS, REGION
+--    remove created* & last_modified* from TRANSCRIPTION_STATUS, TRANSCRIPTION_TYPE, TRANSCRIPTION_URGENCY, TRANSCRIPTION_WORKFLOW
+--    remove workflow_comment from transcription_workflow, instead add trw_id to transcription_comment as foreign key to transcription_workflow
+--    add table transcription_document including foreign key to transcription
+--    replace column tra_id in external_object_directory with trd_id instead, with same change to FK
+--v51 add table annotation_document, its PK, Sequence and FK to annotation
+--    amend external_object_directory to refer to ado_id instead on ann_id
+--    add hea_id to annotation and FK to hearing
+--    adding FKs on ado_id, trd_id on external_object_directory
+--    add is_reporting_restriction to event_handler table
 
 -- List of Table Aliases

```

Comment:
I guess we need to add `node_register` to this list instead of `device_register`?

Reply:
corrected in v52